salvo/README.md

```diff
@@ -0,0 +1,73 @@
+# salvo
```

Comment:
Consistency nit: should "salvo" be capitalized? (below there are instances of that)

Reply:
Fixed